src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -97,6 +97,7 @@ public SimpleInterval(final String str){
          * So we can either keep this parsing code in the constructor or make a static factory method
          * and make multiple objects. We chose the former.
          */
+        Utils.nonNull(str);
```

Comment:
We should probably add a check for empty string as well.  I don't think  the interval from 1 to max-int on conting "" is something that people would ever want.